pkg/flowaggregator/options/options.go

```diff
@@ -61,7 +63,7 @@ func LoadConfig(configBytes []byte) (*Options, error) {
 		return nil, fmt.Errorf("s3Uploader enabled without specifying bucket name")
 	}
 	if !opt.Config.FlowCollector.Enable && !opt.Config.ClickHouse.Enable && !opt.Config.S3Uploader.Enable && !opt.Config.FlowLogger.Enable {
-		return nil, fmt.Errorf("at least one collector / sink should be configured")
+		klog.InfoS("No collector / sink has been configured, so no flow data will be exported")
```

Comment:
Considering it might not be expected in most cases, I suppose this should be warning info?

Reply:
we don't use warning logs any more, there is no `klog.WarningS` function